assets/app/scripts/controllers/overview.js

```diff
@@ -22,6 +20,14 @@ angular.module('openshiftConsole')
     $scope.images = {};
     $scope.imagesByDockerReference = {};
     $scope.podsByServiceByLabel = {};
+    $scope.deploymentPods = {};
+    $scope.servicePodsNoDeployment = {};
+    $scope.solitaryPods = {};
+    $scope.serviceDeploymentsByDeploymentConfig = {};
+    $scope.unservicedDeploymentsByDeploymentConfig = {}
+    $scope.unservicedDeploymentConfigs = {};
+    $scope.serviceDeploymentConfigs = {};
+    $scope.buildPods = {};
```

Comment:
if we can move the logic about which monopods to show and which to hide into the controller, this can become an internal map, not a scope map. either way, add a comment that this is a marker map to keep track of which things are buildpods, not a map of id->object